ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-ranGen.yaml

```diff
@@ -59,5 +59,17 @@ spec:
         disableDrain: true
     - fileName: StorageLV.yaml
       policyName: "local-disks-policy"
+      spec:
+        storageClassDevices:
+          - storageClassName: "example-storage-class-1"
+            volumeMode: Filesystem
+            fsType: xfs
+            devicePaths:
+              - /dev/sdb1
+          - storageClassName: "example-storage-class-2"
+            volumeMode: Filesystem
+            fsType: xfs
+            devicePaths:
+              - /dev/sdb2
```

Comment:
We should tailor the reference PGTs to the expected use case. I don't know how many classes might be typical for a RAN deployment but my guess would be 1. @browsell any insights?

Reply:
From what I recall there is no "typical" storage config; everyone will want something different.